Userland/DevTools/UserspaceEmulator/SoftFPU.cpp

```diff
@@ -0,0 +1,993 @@
+/*
+ * Copyright (c) 2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2020, Leon Albrecht <loen2002.la@gmail.com>
```

Comment:
It's 2021 now.

Reply:
oh.. And apparently I can't spell my mail correctly either